tests/pytest/macros_test.py

```diff
@@ -52,7 +52,13 @@ def copy_and_modify_macro(config_tmpdir, output_tmpdir, init_macro):
             lines = f.readlines()
             for l in lines:
                 l1 = l.split(' ')
-                if l1[0] == '/petalosim/persistency/outputFile':
+                if l1[0] == '/petalosim/persistency/output_file':
+                    l2 = l1[1].split('/')
+                    f_cp.write(l1[0] + ' ' + str(output_tmpdir) + '/' + l2[-1])
+                elif l1[0] == '/Actions/PetAnalysisRunAction/histo_file':
+                    l2 = l1[1].split('/')
+                    f_cp.write(l1[0] + ' ' + str(output_tmpdir) + '/' + l2[-1])
+                elif l1[0] == '/Actions/PetAnalysisRunAction/ntuple_file':
```

Comment:
In this lines I don't see the difference between the if and elif. I think they all do the same.

Reply:
I see this difference, but then, inside the if and the others elif don't they do the same? Maybe I'm not understanding fine the code.

Reply:
An alternative way of doing would be to use just one `if` and `|` to differentiate the options. I'm not sure which way is faster, though.